src/migrations/20190120141524-review.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+module.exports = {
+    up: (queryInterface, Sequelize) => {
```

Comment:
Oh hello 4 space tabs!

We don't have a documented convention, but the rest of the code is 2 spaces for tabs.